### ISSUE TITLE
Improved System Status Report

### ISSUE THREAD

**Issue by jobthomas** [woocommerce/woocommerce#26371]
**Is your feature request related to a problem? Please describe.**
This follows after a discussion with some Happiness Engineers. 

Several elements of the System Status report are useful to have as information, but currently, there's no good filter as to what is more urgent and what is not. 

**Describe the solution you'd like**
Ideally, it'd be great if the system status report works more similar to **Tools > Site Health** where the first tab shows the critical issues and recommended improvements:

![](https://d.pr/i/F3MajB+)
*(If the image does not load, please click this link: https://d.pr/i/F3MajB.)*

The info tab in the above screenshot then contains more details which now are in the WooCommerce System Status. 

**Offer**

I think several of us would be more than willing in helping to flag what are critical issues (such as database version not matching WooCommerce version) and what are recommended things to consider. 

Some of the people who were involved in the other (private discussion): @RiaanKnoetze @iMazed  @jessepearson @amandasjackson 

@mattwondra I think you're interested in this proposal as well.

**Comment by tyxla** [comment]
In addition to what's been suggested above, some good suggestions on anonymization of the system status reports came in as part of #61075.

**Comment by pmcpinto** [comment]
Routing this to engineering for the next step: the report needs technical validation, reproduction, or decomposition.

**Comment by jamesckemp** [comment]
Makes sense to improve system status reports! @beaulebens has also been doing some related work [here](https://github.com/woocommerce/woocommerce/pull/65652).
